JudoPayDotNet/Models/CardDetails.cs

```diff
@@ -43,7 +44,26 @@ public class CardDetails
         /// The type of the card.
         /// </value>
         [DataMember]
+        [Obsolete("Please use CardScheme and CardFunding instead")]
```

Comment:
@James-Lappin  this might be  minor thing, but I think we should have marked this as obsolete a while ago and now we should be removing this completely and adding documentation telling developers this is a breaking change and how to upgrade